resources/views/layouts/dashboard.blade.php

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html lang="en">
```

Comment:
is it a database seeder file or html file ? if it is necessary please remove it.